tcmur_cmd_handler.c

```diff
@@ -2308,6 +2318,312 @@ static int handle_inquiry(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 	return ret;
 }
 
+int get_cdb_length(uint8_t opcode)
+{
+	int cdb_size[8] = {6,10,10,12,16,12,10,10};
+	return cdb_size[((opcode)>>5)&7];
+}
```

Comment:
Why can't you just use tcmu_get_cdb_length?

Reply:
I intend to obtain the cdb_length of REQUESTED OPERATION CODE(cmd->cdb[3]), parameter type of tcmu_get_cdb_length is uint8_t*, meanwhile, data type of REQUESTED OPERATION CODE is uint8_t, therefore, I need to rewrite one function that used to calculate cdb_length.

Reply:
Take the code in the switch in tcmu_get_cdb_length, make a new function out of it and name it something like tcmu_get_cdb_length_by_opcode. Then have tcmu_get_cdb_length call that new function and your code.